src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -32,7 +33,7 @@
 
   @Test
   @Execute(asUser = User.QARENAME)
-  public void UserProvidesCorrectNewNameDoesntClickUnderstandCheckbox() {
+  public void userProvidesCorrectNewNameDoesntClickUnderstandCheckbox() {
     SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage()
         .open()
         .fillFormData("ChesskyTest", "ChesskyTest", "q")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "ChesskyTest" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)